src/Beman/Optional26/tests/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+# cmake-format: off
```

Comment:
Is there a cmake-format problem with the SPDX license identifier?

Reply:
`SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception` is the minimum necessary. Maybe we should just stick with that?

Before:

```CMake
# cmake-format: off
# src/Beman/Optional26/CMakeLists.txt -*-makefile-*-
# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
# cmake-format: on

code
```

After:

```CMake
# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception

code
```

Reply:
If don't mind, I will actually configure clang-format and address this issue in https://github.com/beman-project/Optional26/pull/26. WIP issue.

Reply:
I don't mind if this is handled in another PR. I like the simplicity of my suggestion though.

Reply:
Licensing info is usually at the top in Open Source projects. Bloomberg is the only exception I'm aware of.